schema/auth.json

```
@@ -32,7 +32,8 @@
         "name": {
           "type": "string",
           "description": "Unique auth definition name",
-          "minLength": 1
+          "minLength": 1,
+          "pattern": "^[a-z0-9]([-a-z0-9]*[a-z0-9])$"
```

Per https://github.com/serverlessworkflow/specification/pull/768#discussion_r1208070568 , this should either prohibit "--" at index 2
```suggestion
          "pattern": "^[a-z0-9](([-a-z0-9]){0,2}[a-z0-9])?$|^[a-z0-9][-a-z0-9]([a-z0-9]{2}|-[a-z0-9]|[a-z0-9]-)[-a-z0-9]*[a-z0-9]$"
```
or prohibit any consecutive dashes
```suggestion
          "pattern": "^[a-z0-9](-?[a-z0-9])*$"
```
(here and in all the other analogous regular expressions)